cmd/compose/create.go

```diff
@@ -80,8 +80,8 @@ func createCommand(p *ProjectOptions, dockerCli command.Cli, backend api.Service
 	}
 	flags := cmd.Flags()
 	flags.BoolVar(&opts.Build, "build", false, "Build images before starting containers.")
-	flags.BoolVar(&opts.noBuild, "no-build", false, "Don't build an image, even if it's missing.")
-	flags.StringVar(&opts.Pull, "pull", "missing", `Pull image before running ("always"|"missing"|"never")`)
+	flags.BoolVar(&opts.noBuild, "no-build", false, "Don't build an image, even if it's policy.")
+	flags.StringVar(&opts.Pull, "pull", "policy", `Pull image before running ("always"|"policy"|"never")`)
```

Comment:
We have these args/descriptions across several commands, and I think we need to improve the copy here, especially since we're making changes, as they're pretty vague and unclear as-is.

cc @aevesdocker 

My understanding of desired behaviors:
 * `--build`: Re-build images for all services being launched that have a `build` section. Images with `pull_policy: build` are always re-built unless `--no-build` is specified.
 * `--no-build`: Don't build ANY images, which might mean the project can't start if local versions are not already available in the engine.
 * `--policy`
   * `always`
     * for services with `build` section, fetch the latest base image (`FROM`), similar to `docker build --pull .`
     * for services without a `build` section, fetch the `image` even if it's already present locally (e.g. useful for `:latest`)
   * `never`
      * for services with a `build` section, no-op (AFAIK there's no builder option to say "don't automatically pull missing base images" nor would it make much sense)
      * for services without a `build` section, don't pull the `image` even if it doesn't exist in the engine, which might mean the project can't launch.

Reply:
> `always` for services with build section, fetch the latest base image (FROM), 

Nope, fetch the configured tagged image, which is the result of the build. We don't parse the Dockerfile to know about the base image. No relation with `docker build --pull`